treatment_index.py

```diff
@@ -367,22 +363,335 @@ def create_treatment_points(schema, sde_file):
+    logger.info(f'{schema}.{treatment_index}_points_temp renamed to {schema}.{treatment_index}_points')
 
     # rename backup to temp so we can delete it
-    rename_postgres_table(connection, schema, 'treatment_index_facts_nfpors_points_backup', 'treatment_index_facts_nfpors_points_temp')
-    logger.info(f'{schema}.treatment_index_facts_nfpors_points_backup renamed to {schema}.treatment_index_facts_nfpors_points_temp')
+    rename_postgres_table(cursor, schema, f'{treatment_index}_points_backup', f'{treatment_index}_points_temp')
+    logger.info(f'{schema}.{treatment_index}_points_backup renamed to {schema}.{treatment_index}_points_temp')
 
     # clean up so we can create points again later
-    arcpy.management.Delete(os.path.join(sde_file, f"sweri.{schema}.treatment_index_facts_nfpors_points_temp"))
+    arcpy.management.Delete(os.path.join(sde_file, f"sweri.{schema}.{treatment_index}_points_temp"))
+
+# BEGIN Common Attributes Functions
+
+def add_fields_and_indexes(feature_class, region):
+    new_fields = ('included', 'r2', 'r3', 'r4','r5', 'r6')
+    for field in new_fields:
+        arcpy.management.AddField(feature_class, field, 'TEXT')
+        arcpy.management.AddIndex(feature_class, field, f'{field}_idx_{region}', ascending="ASCENDING")
+
+    arcpy.management.AddIndex(feature_class, 'event_cn', f'event_cn_idx_{region}', unique="UNIQUE", ascending="ASCENDING")
+
+    new_indexes = ('gis_acres', 'activity', 'equipment', 'method')
+    for index in new_indexes: 
+            arcpy.management.AddIndex(feature_class, index, f'{index}_idx_{region}', ascending="ASCENDING")
+       
+def exclude_facts_hazardous_fuels(cursor, schema, table, facts_haz_table):
+    # Do Not Included Entries Already Being Included via Hazardous Fuels
+    cursor.execute('BEGIN;')
+    cursor.execute(f'''
+                   
+    DELETE FROM {schema}.{table}
+    WHERE 
+    event_cn IN(
+        SELECT activity_cn FROM {schema}.{facts_haz_table}
+    )
+          
+    ''')
+    cursor.execute('COMMIT;')
+    logging.info(f"deleted {schema}.{table} entries that are also in FACTS Hazardous Fuels")
+
+def exclude_by_acreage(cursor, schema, table):
+    cursor.execute('BEGIN;')
+    cursor.execute(f'''
+                   
+    DELETE FROM {schema}.{table}
+    WHERE
+    gis_acres <= 5 OR
+    gis_acres IS NULL; 
+    
+    ''')
+    cursor.execute('COMMIT;')
+    logging.info(f"deleted Entries <= 5 acres {schema}.{table}")
+
+def trim_whitespace(cursor, schema, table):
+    cursor.execute('BEGIN;')
+    cursor.execute(f'''
+                   
+    UPDATE {schema}.{table}
+    SET
+    activity = TRIM(activity),
+    method = TRIM(method),
+    equipment = TRIM(equipment);
+    
+    ''')
+    cursor.execute('COMMIT;')
+    logging.info(f"removed white space from activity, method, and equipment in {schema}.{table}")
+
+def include_logging_activities(cursor, schema, table):
```

Comment:
add docstring and brief comment for each rule describing what it does so it's easy to link that logic back to the data paper